src/openforms/registrations/contrib/zgw_apis/plugin.py

```diff
@@ -34,10 +36,60 @@
 from .checks import check_config
 from .client import get_catalogi_client, get_documents_client, get_zaken_client
 from .models import ZGWApiGroupConfig, ZgwConfig
+from .utils import process_according_to_eigenschap_format
 
 logger = logging.getLogger(__name__)
 
 
+class VariablesProperties(TypedDict):
+    component_key: str
+    eigenschap: str
+
+
+def get_property_mappings_from_submission(
+    submission: Submission, connections: list[VariablesProperties]
```

Comment:
```suggestion
    submission: Submission, mappings: list[VariablesProperties]
```
please consistently use the "mapping" variable name(s)